statsmodels/sandbox/stats/multicomp.py

```diff
@@ -176,6 +176,25 @@ def get_tukeyQcrit2(k, df, alpha=0.05):
     return qsturng(1-alpha, k, df)
 
 
+def get_tukeyPval(k, df, q):
```

Comment:
pep-8 names (no capital letters and no camel case in function names)
e.g. ` get_tukey_pvalue`